mon-pix/app/routes/login-pe.js

```diff
@@ -0,0 +1,49 @@
+import Route from '@ember/routing/route';
+import OIDCAuthenticationRouteMixin from 'ember-simple-auth-oidc/mixins/oidc-authentication-route-mixin';
+
+import getAbsoluteUrl from 'ember-simple-auth-oidc/utils/absoluteUrl';
+
+import { v4 } from 'uuid';
+import config from 'ember-simple-auth-oidc/config';
+
+const { host, clientId, authEndpoint, loginHintName } = config;
+
+export default class LoginPeRoute extends Route.extend(OIDCAuthenticationRouteMixin) {
+
+  _handleRedirectRequest(queryParams) {
+    const scope = `application_${clientId}%20api_peconnect-individuv1%20openid%20profile%20email`;
+
+    const state = v4();
+
+    this.session.set('data.state', state);
+
+    /**
+     * Store the `nextURL` in the localstorage so when the user returns after
+     * the login he can be sent to the initial destination.
+     */
+    if (!this.session.get('data.nextURL')) {
+      this.session.set(
+        'data.nextURL',
+        this.session.get('attemptedTransition.intent.url'),
+      );
+    }
+
+    const key = loginHintName || 'login_hint';
+
+    const search = [
+      `client_id=${clientId}`,
+      `redirect_uri=${this.redirectUri}`,
+      'response_type=code',
+      `state=${state}`,
+      `scope=${scope}`,
+      queryParams[key] ? `${key}=${queryParams[key]}` : null,
+    ]
+      .filter(Boolean)
+      .join('&');
+
+    const updatedAuthEndpoint = `${authEndpoint}?realm=%2Findividu`;
```

Comment:
En cherchant la configuration authEndpoint dans le code de l'addon, c'est utilisé une seule fois dans la redirection.

On vous propose de ne pas override la fonction de lib si on peut seulement changer la configuration de l'endpoint en rajoutant le query param du realm dans la configuration oidc

authEndpoint=/connexion/oauth2/authorize?realm=%2Findividu

Reply:
le `?realm` n'est pas compatible avec `this._redirectToUrl(`${getAbsoluteUrl(host)}${authEndpoint}?${search}`);`
contenu dans la fonction `_handleRedirectRequest(queryParams)` d'origine : 
https://github.com/adfinis-sygroup/ember-simple-auth-oidc/blob/53356814f631e63e44d8504803fb98dd7834d779/addon/mixins/oidc-authentication-route-mixin.js#L108